app/src/main/res/values/strings.xml

```diff
@@ -105,22 +105,22 @@
     <string name="need_to_talk_message">I need to talk. Message sent through First Aide\'s Circle of Trust</string>
     <string name="msg_sent">Message sent</string>
     <string name="ok">OK</string>
-    <string name="message_failed">Message Sending Failed for Comrade </string>
+    <string name="message_failed">Message Sending Failed for Contact </string>
```

Comment:
Contact should be 'contact' . Sorry about not noticing that with the "Comrades". Do not change the capitalized Emergency Contacts. Change all the other locations accordingly. If you have any doubt ask here or in the mailing list.